### ISSUE TITLE
PR are sometimes wrongly associated with issue child of #2909

### ISSUE THREAD

**Issue by pvdlg** [waffleio/waffle.io#3613]
The issue https://github.com/semantic-release/semantic-release/issues/65 appeared to be linked to the PR https://github.com/semantic-release/semantic-release/pull/860.

This is because the PR contains the following in its body:
```md
....

<code>Revert "Fix #65. Check if the parsed resource is not null."</code>

....
```

Greenkeeper opened this PR which list the commits since the last release of a dependency. It turns out this dependency had a commit with the message `Fix #65. Check if the parsed resource is not null`. However the `#65` here has nothing to do with https://github.com/semantic-release/semantic-release/issues/65.

The problem is that Waffle detects `#65` in the body of the PR, however it shouldn't as it is between `<code>` and `</code>`.

You can refer to [issue-parser](https://github.com/pvdlg/issue-parser) that [handle this case](https://github.com/pvdlg/issue-parser#ignore-references-in-code-tags) if that helps.

**Comment by pvdlg** [comment]
Actually, I think it can probably be closed as a duplicate of #2909. Although the example in this issue might be helpful.

**Comment by AnaMSteele** [comment]
@pvlg closing as duplicate based on your comment above. Thank you.
